src/api-reference/request/v4.endpoints.markdown

```diff
@@ -25,33 +25,34 @@ If a new or existing project please use version [3.0](./v3.request.html) of this
 * [Travel Agency Resources](#travel-agency-resource)
   * [Get the description of a travel agency office](#get-travel-agency-description-resource)
 * [Schema](#schema)
-  * [Request](#schema-request)
-  * [ApprovalStatus](#schema-approvalstatus)
-  * [Customfield](#schema-customfield)
-  * [Location](#schema-location)
-  * [Amount](#schema-amount)
-  * [ResourceLink](#schema-resourcelink)
-  * [RequestLink](#schema-requestlink)
-  * [Link](#schema-link)
-  * [Expense](#schema-expense)
-  * [Allocation](#schema-allocation)
-  * [Exchange Rate](#schema-exchangerate)
-  * [Expense Type](#schema-expensetype)
-  * [Travel Allowance](#schema-travelallowance)
-  * [Trip Data](#schema-tripdata)
-  * [Vendor](#schema-vendor)
-  * [Segment Leg](#schema-segmentleg)
-  * [Segment Type](#schema-segmenttype)
+  * [Request](#request)
+  * [ApprovalStatus](#approvalstatus)
+  * [Customfield](#customfield)
+  * [List of Requests](#listofrequests)
+  * [Location](#location)
+  * [Amount](#amount)
+  * [ResourceLink](#resourcelink)
+  * [RequestLink](#requestlink)
+  * [Link](#link)
+  * [Expense](#expense)
+  * [Allocation](#allocation)
+  * [Exchange Rate](#exchangerate)
+  * [Expense Type](#expensetype)
+  * [Travel Allowance](#travelallowance)
+  * [Trip Data](#tripdata)
+  * [Vendor](#vendor)
+  * [Segment Leg](#segmentleg)
+  * [Segment Type](#segmenttype)
 
 ## HTTP Status Codes and Headers
 
-To learn more about request headers and response HTTP status codes and headers for this API see [Travel Request v4 - HTTP Status Codes](./v4.response-codes.html).
+To learn more about request headers and response HTTP status codes and headers for this API see [Travel Request v4 - HTTP Status Codes](./v4.response-codes.html) and [Travel Request v4 - Headers](./v4.headers.html).
```

Comment:
@laurentlhermine can you delete the "and [Travel Request v4 - Headers](./v4.headers.html)" part and delete "Travel" in the "Travel Request v4 HTTP Status Codes" link?

Reply:
@charlieconcur as per below comment : we have closed the sprint and do not have time currently to work on this page. This might be part of the next monthly release (other modification of the documentation will be done during next monthly sprint to manage the sorting of results). For the monthly release planned this Saturday we will not be able to manage it, if maintained in the documentation, might bring confusion